test/core/V3/GenArt721CoreV3_ProjectConfigure.test.ts

```diff
@@ -754,6 +754,40 @@ describe("GenArt721CoreV3 Project Configure", async function () {
       expect(script).to.equal(SQUIGGLE_SCRIPT);
     });
 
+    it("uploads chromie squiggle script, and then purges it", async function () {
```

Comment:
Reminder: TODO - add additional tests that ensure external accounts cannot trigger selfdestruct

Reply:
Done in https://github.com/ArtBlocks/artblocks-contracts/pull/304/commits/0d6d941a92507311e39d839129c2ca9c4d4cd61f

Still adding test coverage for one other critical path here and then will take this PR out of draft mode.